allocate.go

```diff
@@ -337,6 +346,13 @@ func Flag(name string, value interface{}) ExecAllocatorOption {
 	}
 }
 
+// Env is a generic environment variable to pass into Chrome.
```

Comment:
You should document that this is extra vars on top of the current process' environment, i.e. `os.Environ()`.

Reply:
Done as well!